configs/Kconfig

```diff
@@ -113,4 +117,10 @@ config DEMO_ANIMATION
     bool "Build animation demo"
     default y
     depends on DEMO_APPLICATIONS 
+
+config DEMO_IMAGE
+    bool "Build image demo"
```

Comment:
Since the demo program is dedicated to scalar images, its description should be refined to "Build scalable image demo."